presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -198,6 +200,17 @@ public void testFromUnixTimeWithOffset()
         assertFunction("from_unixtime(" + seconds + ", " + timeZoneHoursOffset + ", " + timezoneMinutesOffset + ")", TIMESTAMP_WITH_TIME_ZONE, toTimestampWithTimeZone(expected));
     }
 
+    @Test
+    public void testFromUnixTimeWithOffsetNegative()
+    {
+        DateTime dateTime = new DateTime(2001, 1, 22, 3, 50, 5, 0, DATE_TIME_ZONE);
+        double seconds = dateTime.getMillis() / 1000.0;
```

Comment:
You could use constant `seconds` (even within the text `"from_unixtime(...` below).

Reply:
Sure done.